docs/sdk/authentication/session-sigs/get-pkp-session-sigs.md

```diff
@@ -0,0 +1,148 @@
+
+let litNodeClient: LitNodeClient;
+
+litNodeClient = new LitNodeClient({
+      litNetwork: LitNetwork.DatilTest,
+      debug: false,
+    });
+await litNodeClient.connect();
+```
+
+### Instantiating a `LitContracts` Instance
+Here we are initializing an instance of `LitContracts`. This allows us to interact with smart contracts on the Lit network. 
+
+```ts
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import { LitNetwork } from "@lit-protocol/constants";
+
+const litContracts = new LitContracts({
+    signer: ethersSigner,
+    network: LitNetwork.DatilTest,
+    debug: false,
+});
+await litContracts.connect();
+```
+
+### Generating Session Signatures
+In this example, we're granting the capability to request to decrypt any data that we may be authorized to decrypt (i.e. we satisfy the Access Control Conditions the data was encrypted with). We could, however, specify the [LitAccessControlConditionResource](https://v6-api-doc-lit-js-sdk.vercel.app/classes/auth_helpers_src.LitAccessControlConditionResource.html) for specific encrypted data we're permitting the decryption capability for. In real-world applications, it's more common and secure to limit access to specific Lit resources instead of specifying the wildcard (`"*"`) identifier.
+
+To get the Lit resource identifier for other resources, you can use the other methods included in [@lit-protocol/auth-helpers](https://v6-api-doc-lit-js-sdk.vercel.app/modules/auth_helpers_src.html) package.
+
+If you would like to use this function on the `datil` or `datil-test` networks, a `capacityDelegationAuthSig` is required. Please also keep in mind that implementing this requires owning or minting a PKP and some form of authentication (e.g. a custom Lit Action, Auth Method, or AuthSig). How this is done can be found in the full code example.
+
+
+```ts
+import { LitAbility, LitPKPResource } from "@lit-protocol/auth-helpers";
+
+const sessionSignatures = await litNodeClient.getPkpSessionSigs({
+    pkpPublicKey: pkp.publicKey!,
+    capabilityAuthSigs: [capacityDelegationAuthSig],
+    authMethods: [authMethod],
+    resourceAbilityRequests: [
+        {
+            resource: new LitPKPResource("*"),
+            ability: LitAbility.PKPSigning,
+        },
+    ],
+    expiration: new Date(Date.now() + 1000 * 60 * 10).toISOString(), // 10 minutes
+});
+```
+
+## Clearing Local Storage
+
+If you want to clear the session key stored in the browser local storage, you can call the [`disconnectWeb3` method](https://js-sdk.litprotocol.com/functions/auth_browser_src.ethConnect.disconnectWeb3.html).
+
+## Summary
+The full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/tree/master/session-signatures/getPkpSessionSigs). 
+
+After executing the example implementation above, you will have generated session signatures that allow you to request decrypting data that you have satisfied the Access Control Conditions for.
```

Comment:
Note: The last sentence of `getPkpSessionSigs` is wrong, it will be fixed in the next push